zombienet_tests/parachains/0001-parachains-smoke-test.toml

```diff
@@ -10,10 +10,26 @@ command = "polkadot"
   name = "alice"
   extra_args = [ "--alice" ]
 
+  [[relaychain.nodes.env]]
+  name = "RUST_LOG"
+  value = "parachain=debug"
+
+#  [[relaychain.nodes.overrides]]
+#  local_path = "../../target/release/polkadot"
+#  remote_name = "wonderland"
```

Comment:
Hi @drahnr, just a comment on this config. `Zombienet` use remote_name to create the `file` in the container, in this case `wonderland` **but** this config **use** the default command (e.g `polkadot`) `so` wonderland will not be used.

Related issue in `zombienet` https://github.com/paritytech/zombienet/issues/46 to explore make this optional.
Thx!

Reply:
In that case, `zombienet` should point me to precisely this conflict with line numbers :upside_down_face:

Reply:
Sounds good! I open this [issue](https://github.com/paritytech/zombienet/issues/57) to track this.
Thanks!